[2023-03-01 13:10:43,006] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-09T13:05:00+00:00 [queued]>
[2023-03-01 13:10:43,299] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-09T13:05:00+00:00 [queued]>
[2023-03-01 13:10:43,301] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:10:43,303] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 13:10:43,304] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:10:43,577] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-12-09T13:05:00+00:00
[2023-03-01 13:10:43,605] {standard_task_runner.py:52} INFO - Started process 20725 to run task
[2023-03-01 13:10:43,719] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-12-09T13:05:00+00:00', '--job-id', '4969', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmp2fg365nk', '--error-file', '/tmp/tmp7zxlutjn']
[2023-03-01 13:10:43,727] {standard_task_runner.py:77} INFO - Job 4969: Subtask get_name
[2023-03-01 13:10:44,565] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-12-09T13:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 13:10:45,094] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-12-09T13:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-09T13:05:00+00:00
[2023-03-01 13:10:45,327] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 13:10:45,480] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221209T130500, start_date=20230301T131043, end_date=20230301T131045
[2023-03-01 13:10:46,101] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-03-01 13:10:46,208] {local_task_job.py:146} INFO - Task exited with return code 0
